src/App.js

```diff
@@ -1,23 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
```

Comment:
I think that this label can be removed

Reply:
ready

Reply:
Thanks @valerialp. I removed it. Please let me know if you approve the pulll request to proceed :)

Reply:
Good, you can merge.

Reply:
great!